db/seeds.rb

```diff
@@ -5,3 +5,46 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'faker'
+STATE_ARRAY = ["AL", "AK", "AZ", "AR", "CA", "CO", "CT", "DE", "FL", "GA",
```

Comment:
You are using this array in a couple of different places. It probably makes sense to put it into a module or a class.

Read over the [article we have on seeding](https://learn.launchacademy.com/lessons/seeding). I think it will help you out here.